### PR TITLE
Implemented ability to open packed projects in studio

### PR DESCRIPTION
**Pull request by glebmish**

### CODE REVIEW

File: src/appleseed.studio/mainwindow/mainwindow.cpp [hunk 1]
```diff
@@ -1122,7 +1122,7 @@ void MainWindow::slot_open_project()
         get_open_filename(
             this,
             "Open...",
-            "Project Files (*.appleseed);;All Files (*.*)",
+            "Project Files (*.appleseed);;Project Packed Files (*.appleseedz);;All Files (*.*)",
```
Comment (dictoon) [inline src/appleseed.studio/mainwindow/mainwindow.cpp]: *Packed Project Files* would probably be better.

Comment (glebmish) [inline src/appleseed.studio/mainwindow/mainwindow.cpp]: I'll change it later. Not at home already

Comment (dictoon) [inline src/appleseed.studio/mainwindow/mainwindow.cpp]: No worries, I can fix it. Thanks man!

Comment (glebmish) [inline src/appleseed.studio/mainwindow/mainwindow.cpp]: Thanks!

**Pull request merged**